ansible/playbooks/SQLclusterPlaybooks/step3-rhazure.yml

```diff
@@ -0,0 +1,125 @@
+- name: Configure a Microsoft SQL Server AG
+  hosts: all
+  vars:
```

Comment:
I recommend pulling the vars out of the playbook `vars` sections and move them either into the inventory, `group_vars/all.yml` or in a dedicated vars file, especially for variables that are used in all three playbooks.

It will make the playbooks cleaner to read and make it easier to change configuration.

Reply:
I need to think further on these changes, but will likely update this to use vars_files in the future for consistency.